src/main/java/webserver/RequestHeader.java

```diff
@@ -0,0 +1,24 @@
+package webserver;
+
+import java.util.Map;
+
+public class RequestHeader extends Header {
+    private static final String METHOD_KEY = "method";
+    private static final String PATH_KEY = "path";
+
+    public RequestHeader(Map<String, String> attributes) {
+        super(attributes);
+    }
```

Comment:
생성자의 접근 제어자를 protected로 변경할 수 있음
```suggestion
    protected RequestHeader(Map<String, String> attributes) {
        super(attributes);
    }
```